staging/src/k8s.io/kubectl/pkg/cmd/debug/profiles.go

```diff
@@ -210,6 +245,10 @@ func (p *netadminProfile) Apply(pod *corev1.Pod, containerName string, target ru
 		useHostNamespaces(pod)
 
 	case podCopy:
+		p.keeps.removeLabels(pod)
```

Comment:
and this flags are only applicable for copying pod debug type and not relevant for ephemeral debug and node debug. I'm thinking maybe we should mention in the flags descriptions to this.

Reply:
I've added the flags descriptions like "This flag only works when used with '--copy-to'".